tools/bionano/bionano_scaffold.xml

```diff
@@ -60,7 +60,7 @@
             && python '$__tool_directory__/remove_fake_cut_sites.py' $output_file_NCBI 'SCAFFOLD_NCBI_trimmed.fasta' 'output.log'
             && python '$__tool_directory__/remove_fake_cut_sites.py' $output_file_not_scaffolded 'NOT_SCAFFOLDED_trimmed.fasta' 'output.log'
         #end if
-        && echo >> hybrid_scaffolds/*cut.txt
+        && touch hybrid_scaffolds/*cut.txt
```

Comment:
will this not create a `*cut.txt` file?

Reply:
Yes, it will create if it doesn't exist.

Reply:
but the star is part of the filename ...
And as far as I can see this is what caused the crash I have sent you...

why not `cat hybrid_scaffolds/*cut.txt '$key_file'` ... without touch and and mv at all.